src/ogc/wcs/WebCoverageService.js

```diff
@@ -35,27 +35,67 @@ define([
         /**
          * Represents a Web Coverage Service and provides functionality for interacting with the service. Includes
          * functionality for retrieving DescribeCoverage documents and providing WCS version agnostic coverage objects.
-         * @param serviceAddress the url of the Web Coverage Service
          * @constructor
          */
-        var WebCoverageService = function (serviceAddress) {
+        var WebCoverageService = function () {
+
+            /**
+             * The URL for the Web Coverage Service
+             */
+            this.serviceAddress = null;
 
             /**
              * A collection of the coverages available from this service. Not populated until service is initialized by
              * the connect method.
              * @type {Array}
              */
             this.coverages = [];
+
+            /**
+             * The WCS GetCapabilities document for this service.
+             * @type {WcsCapabilities}
+             */
+            this.capabilities = null;
+
+            /**
+             * A map of the coverages to their corresponding DescribeCoverage documents.
+             * @type {{}}
+             */
+            this.coverageDescriptions = {};
         };
 
         /**
-         * Connects to the Web Coverage Service specified in the constructor. This function handles version negotiation
-         * and capabilities document retrieval. The return is a Promise which returns the initialized
-         * WebCoverageService.
-         * @returns {Promise} a Promise of a WebCoverageService
+         * An array of compatible WCS versions supported by this client service.
+         * @type {string[]}
+         */
+        WebCoverageService.COMPATIBLE_WCS_VERSIONS = ["1.0.0", "2.0.0", "2.0.1"];
+
+        /**
+         * The maximum length of a constructed DescribeCoverage url request.
+         * @type {number}
+         */
+        WebCoverageService.MAX_URL_LENGTH = 2083;
+
+        /**
+         * Contacts the Web Coverage Service specified by the service address. This function handles version negotiation
+         * and capabilities and describe coverage document retrieval. The return is a Promise to a fully initialized
+         * WebCoverageService which includes an array of WcsCoverage objects available from this service.
+         * @param serviceAddress the url of the WebCoverageService
+         * @returns {PromiseLike<WebCoverageService>}
          */
-        WebCoverageService.prototype.connect = function () {
+        WebCoverageService.create = function (serviceAddress) {
```

Comment:
I jumped between `load` and `create` and settled on `create` because of how it looks when a client invokes it. It seems to make sense:
```
WebCoverageService.create("https://worldwind26.arc.nasa.gov/wcs")
    .then(function (wcs) {
       wcs.coverages...
});
```

I think there is a potential for confusion on creating a Web Coverage Service, but discover, init, load, didn't seem to clearly explain we are creating our client side WebCoverageService for interaction with a Web Coverage Service.